Resources/views/ContentType/view_content_type.html.twig

```diff
@@ -10,99 +10,151 @@
         {link: path('admin_contenttypeGroupView', {'contentTypeGroupId': content_type_group.id}), label: content_type_group.identifier},
         {link: null, label: content_type_name}
     ] %}
     {{ parent() }}
 {% endblock %}
 
 {% block header_title %}
-    <h1 class="ez-page-header-name" data-icon="&#xe61a;">{{ content_type_name }} [{{ 'content_type.content_count'|transchoice(content_count) }}]</h1>
+    <h1 class="ez-page-header-name ez-contenttype-icon ez-contenttype-icon-{{ content_type.identifier }}">{{ content_type_name }}</h1>
 {% endblock %}
 
 {% block content %}
     <section class="ez-serverside-content">
-
-        <p class="ez-technical-infos">
-            {{
-                "content_type.last_modified"|trans({
-                    "%date%": content_type.modificationDate|localizeddate(locale=app.request.locale),
-                    "%modifier%": modifier.login
-                })
-            }}
-        </p>
-
-        <div class="pure-g">
-            <div class="pure-u-1-2">
-                <div>
-                    <h6>{{ "content_type.name"|trans }}</h6>
-                    {{ content_type_name }}
+        <section class="ez-contenttype-view">
+            <p class="ez-technical-infos">
+                {{
+                    "content_type.last_modified"|trans({
+                        "%date%": content_type.modificationDate|localizeddate(locale=app.request.locale),
+                        "%modifier%": modifier.login
+                    })|desc("Last modified: %date% by %modifier%")
+                }}
+            </p>
+
+            <section>
+                <h2 class="ez-contenttype-title">{{ 'content_type.metadata.title'|trans|desc("Global properties")}}</h2>
+                <div class="ez-contenttype-metadata">
+                    <div class="ez-contenttype-metadata-name">{{ "content_type.name"|trans|desc("Name") }}</div>
+                    <div class="ez-contenttype-metadata-value">{{ content_type_name }}</div>
                 </div>
-
-                <div>
-                    <h6>{{ "content_type.identifier"|trans }}:</h6>
-                    {{ content_type.identifier }}
+                <div class="ez-contenttype-metadata">
+                    <div class="ez-contenttype-metadata-name">{{ "content_type.identifier"|trans|desc("Identifier") }}</div>
+                    <div class="ez-contenttype-metadata-value">{{ content_type.identifier }}</div>
                 </div>
-
-                <div>
-                    <h6>{{ "content_type.description"|trans }}:</h6>
-                    {{ content_type_description }}
+                <div class="ez-contenttype-metadata">
+                    <div class="ez-contenttype-metadata-name">{{ "content_type.description"|trans|desc("Description") }}</div>
+                    <div class="ez-contenttype-metadata-value">{{ content_type_description }}</div>
```

Comment:
We need to cover the bug with the `description` fields.

I suggest we simply drop that line for the time being, and add it when it fixed.